src/components/datepicker/calendar.spec.js

```diff
@@ -66,16 +66,30 @@ describe('md-calendar', function() {
     return null;
   }
 
+  /** Find the `tbody` for a year in the calendar. */
+  function findYearElement(year) {
+    var years = element.querySelectorAll('[md-calendar-year-body]');
+    var yearHeader = year.toString();
```

Comment:
Can we not stamp the year element with an attribute to more directly query it, similar to how the date cells operate?